javaparser-core/src/main/java/com/github/javaparser/PositionUtils.java

```diff
@@ -105,11 +106,8 @@ private static int beginColumnWithoutConsideringAnnotation(Node node) {
     }
 
     private static Node beginNodeWithoutConsideringAnnotations(Node node) {
-        if (node instanceof MethodDeclaration) {
-            MethodDeclaration casted = (MethodDeclaration) node;
-            return casted.getType();
-        } else if (node instanceof FieldDeclaration) {
-            FieldDeclaration casted = (FieldDeclaration) node;
+        if (node instanceof MethodDeclaration || node instanceof FieldDeclaration) {
```

Comment:
Is that correct? It won't be equivalent to the old code as there are more things implementing `TypedNode` than `Method`/`FieldDeclaration`.